yt_dlp/extractor/gedidigital.py

```diff
@@ -1,198 +1,105 @@
-import re
-
 from .common import InfoExtractor
-from ..utils import (
-    base_url,
-    determine_ext,
-    int_or_none,
-    url_basename,
-    urljoin,
-)
+from ..utils import int_or_none, join_nonempty, js_to_json, mimetype2ext
 
 
 class GediDigitalIE(InfoExtractor):
-    _VALID_URL = r'''(?x:(?P<base_url>(?:https?:)//video\.
+    _VALID_URL = r'''(?x:(?:https?:)//(?:\w+\.)?
```

Comment:
this can be cleaned up but I can't make a suggestion until I see how embed extraction is (re-)implemented

Reply:
I think I'll reimplement the embed extraction like it was before. I can't find any webpage that embeds gedi videos but looks like the urls with `/embed` are still valid so the logic should still be the same

Reply:
The embed extraction was re-added